### PR TITLE
chore: update upgrade e2e

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

CHT recently released the new nav; this PR updates the unit tests to run the upgrade from `latest` CHT version (with new nav) instead of old nav. 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:upgrade-tests-after-new-nav-release/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:upgrade-tests-after-new-nav-release/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:upgrade-tests-after-new-nav-release/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/e2e/upgrade/upgrade.wdio-spec.js
Comment (latin-panda) [inline tests/e2e/upgrade/upgrade.wdio-spec.js:120]: @tatilepizs I reduced the usage of these weird IFs, at the end, these are same asserts but clearer.

**Comment by latin-panda** [comment]
@tatilepizs, the upgrade tests are already working in my local environment for `4.2.1` and `latest`. I am just waiting for the CI. 
You can review it now :) Thanks!

**Comment by latin-panda** [comment]
@tatilepizs All checks passed! 🤩 
If you approve it, please merge it to unblock the PRs of other devs

File: tests/e2e/upgrade/upgrade.wdio-spec.js
Comment (tatilepizs) [inline tests/e2e/upgrade/upgrade.wdio-spec.js:75]: 🌟

**Review by tatilepizs — APPROVED** [review]
Thank you! 🌟

**Pull request merged**
